src/utils/got.ts

```diff
@@ -1,9 +1,47 @@
-import got, { ExtendOptions } from 'got'
+import got, { ExtendOptions, TimeoutError } from 'got'
+import { logger } from '../log'
 
 // https://github.com/sindresorhus/got/blob/HEAD/documentation/quick-start.md#options
 const options: ExtendOptions = {
   timeout: {
-    request: 10 * 1000,
+    request: 15 * 1000,
+  },
+  hooks: {
+    beforeError: [
+      (err) => {
+        if (err instanceof TimeoutError) {
+          logger.warn(
+            {
+              err,
+              request: {
+                url: err.request.requestUrl,
+                options: err.request.options,
+                timings: err.timings,
+              },
+            },
+            `Request timed out on ${err.request.options.method} ${err.request.requestUrl}`,
+          )
+        }
+        return err
```

Comment:
Do you think we should extend our custom got error serialization in https://github.com/valora-inc/logging/blob/6258d46cef6b04ed4b2ba42bd7fe000c55aacd44/src/logging.ts#L240-L249 to include timings too? (url was already there from what I remember).

Reply:
yep that seems like a good idea